src/components/Card/Card.js

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import getMergedClassNames from '../../utils/getMergedClassNames';
+import styles from './style.scss';
+
+const baseClass = 'card';
+
+const Card = ({ img, title, children }) => {
+  const hasComponents = React.Children.count(children) >= 1;
+  const mergedClassNames = getMergedClassNames(styles[baseClass]);
+  const titleClass = classNames({
+    [`${baseClass}__title`]: hasComponents,
+    [`${baseClass}__empty`]: !hasComponents
+  });
+  return (
+    <div className={mergedClassNames}>
+      <div className={styles[titleClass]}>
+        {img && <img src={img} className={styles[`${baseClass}__img`]} />}
```

Comment:
there is no way to add e.g alt to this image

Reply:
should I provide all img props or only alt?

Reply:
forget about it :P there is no need. I wonder why we are forcing img usage here. Wouldn't it be useful to use svg here?

Reply:
In agent app we handle images, changing it now can causes issues. But I can extend this with svg, what do you think?

Reply:
https://caniuse.com/#feat=svg-img svg in image tag is widely supported but i thing it will require some styling. Can we add `<img {...imgProps} src={img} className={cx(styles[`${baseClass}__img`], {[imgProps.className] :imgProps.className })} />` ?

Reply:
I checked it, it doesn't require additional styling. Example `src="https://placeholder.pics/svg/300x300/30FFFF-9458FF"`
![image](https://user-images.githubusercontent.com/24878280/59019398-44837700-8848-11e9-8a72-b40d1dffce00.png)

Reply:
ok, let's omit those `imgProps`, 'cause this implementation is sufficient for now :)